stable/database/values.yaml

```diff
@@ -198,6 +198,7 @@ database:
     # All time values are in seconds unless the unit is included in the name.
     hotCopy:
       enablePod: true
+      enableBackups: true
```

Comment:
What would be the preferred default, enabled or disabled? for defense I think we should follow the preferred default in case enableBackup is not supplied.

Thanks for reviewing.

Reply:
Enabling journal hot copy without a backup job will slowly accumulate journal disk space and shut down the SM. As such, I have a preference for enabling backup jobs by default. It is a more safe option. @acabrele?

Reply:
Yes agree to default on.  I guess there is also still the other possible invalid scenario where database.sm.hotCopy.enableBackups could be set to false but journalBackup.enabled could be true leading to no backups being taken against a hotcopy enabled SM.  We'd need to add a check for database.sm.hotCopy.enableBackups=true to the nuosm start options which currently only checks for journalBackup.enabled